### PR TITLE
add release notes and doc for smart-joining views

### PR DESCRIPTION
**Pull request by ObiWahn**

### CODE REVIEW

File: 3.6/release-notes-new-features36.md [hunk 1]
```diff
@@ -9,3 +9,14 @@ The following list shows in detail which features have been added or improved in
 ArangoDB 3.6. ArangoDB 3.6 also contains several bug fixes that are not listed
 here.
 
+AQL
+---
+
+### SmartJoins for Views
+
+In AQL queries views are now considered for SmartJoins. All rules for
+collections apply unchanged. A view to qualifies for a SmartJoin when the
```
Comment (jsteemann) [inline 3.6/release-notes-new-features36.md]:
```suggestion
collections apply unchanged. A view qualifies for a SmartJoin if the
```

File: 3.6/release-notes-new-features36.md [hunk 2]
```diff
@@ -9,3 +9,14 @@ The following list shows in detail which features have been added or improved in
...
+following conditions are met. All collections forming the view are sharded
```
Comment (jsteemann) [inline 3.6/release-notes-new-features36.md]:
```suggestion
following conditions are met: all collections forming the view are sharded
```

File: 3.6/release-notes-new-features36.md [hunk 3]
```diff
@@ -9,3 +9,14 @@ The following list shows in detail which features have been added or improved in
...
+equally. One of those collections is picked to represent the view and must
+fulfill the already know requirements with the join partner. The partner may be
```
Comment (jsteemann) [inline 3.6/release-notes-new-features36.md]:
```suggestion
fulfill the already known requirements with the join partner. The partner may be
```

File: 3.6/release-notes-new-features36.md [hunk 4]
```diff
@@ -9,3 +9,14 @@ The following list shows in detail which features have been added or improved in
...
+a collection or an other view.
```
Comment (jsteemann) [inline 3.6/release-notes-new-features36.md]:
```suggestion
a collection or another view.
```

File: 3.6/smartjoins.md [hunk 5]
```diff
@@ -287,15 +287,18 @@ Limitations
 
 The SmartJoins optimization is currently triggered only for data selection queries,
 but not for any data-manipulation operations such as INSERT, UPDATE, REPLACE, REMOVE
-or UPSERT, neither traversals, subqueries or views.
+or UPSERT, neither traversals or subqueries.
 
-It will only be applied when joining two collections with an identical sharding setup. 
-This requires the second collection to be created with its *distributeShardsLike* 
-attribute pointing to the first collection.
+It will only be applied when joining collections or vies with an identical
```
Comment (jsteemann) [inline 3.6/smartjoins.md]:
```suggestion
It will only be applied when joining collections or views with an identical
```

File: 3.6/smartjoins.md [hunk 6]
```diff
@@ -287,15 +287,18 @@ Limitations
...
+sharding setup. This requires all involved but one collection to be created
+with its *distributeShardsLike* attribute pointing to the collection that is
+the exception. All collections forming a view must be shared in the same way
```
Comment (jsteemann) [inline 3.6/smartjoins.md]:
```suggestion
the exception. All collections forming a view must be sharded in the same way,
```

**Comment by Simran-B** [comment]
Should add an example using SmartJoins with View + collection as well as View + View

**Pull request merged**